tests/unit/account_balance_query_test.py

```
@@ -28,28 +36,145 @@ def test_execute_account_balance_query():
     response_sequences = [[balance_response]]
-    
+
     # Use the context manager to set up and tear down the mock environment
     with mock_hedera_servers(response_sequences) as client:
         # Create the query and verify no exceptions are raised
         try:
-            CryptoGetAccountBalanceQuery().set_account_id(AccountId(0, 0, 1800)).execute(client)
+            CryptoGetAccountBalanceQuery().set_account_id(
+                AccountId(0, 0, 1800)
+            ).execute(client)
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)